src/python/rebalance/rebalance_rackaware.py

```diff
@@ -490,29 +486,34 @@ def generate_reassignment_plan(plan_directory, topics, brokers_info, compute_sto
 
             logger.debug("Start with position in Rack Alternated List: %s", str(next_Leader))
 
-            reassignment_plan, balanced_partitions_for_topic = rassignment_Generator._generate_reassignment_plan_for_topic(replica_count_topic, next_Leader, rack_alternated_list, fd_count, ud_count, brokers_replica_count, force_rebalance)
-            logger.debug("Already balanced partitions for the topic %s are: %s", topic, balanced_partitions_for_topic)
+            reassignment_plan, balanced_partitions_for_topic = reassignment_generator._generate_reassignment_plan_for_topic(replica_count_topic, next_Leader, rack_alternated_list, fd_count, ud_count, brokers_replica_count, force_rebalance)
+            logger.debug("\nAlready balanced partitions for the topic %s are: \n%s", topic, balanced_partitions_for_topic)
 
             for partition in balanced_partitions_for_topic:
                 global_balanced_partitions.append(partition)
             if reassignment_plan is not None:
-                logger.info("Generated reassignment plan for topic: %s", topic)
+                logger.info("\n\nReassignment plan generated for topic: %s\n%s\n", topic, reassignment_plan)
```

Comment:
The same here. Is "\n\n" intented? Every log message should have started on a new line.

Reply:
yup. These are important logs hence wanted to add the \n here